src/database/populate.py

```
@@ -213,63 +218,49 @@
-        """
-        Build a list of dictionaries representing movie records to be inserted into MovieModel.
-
-        :param data: The preprocessed DataFrame.
-        :param country_map: A mapping of country codes to CountryModel instances.
-        :return: A list of dictionaries, each representing a new movie record.
-        """
         movies_data: List[Dict[str, object]] = []
         for _, row in tqdm(
             data.iterrows(), total=data.shape[0], desc="Processing movies"
         ):
             country = country_map[row["country"]]
```

Ruff (0.11.9)

Local variable `country` is assigned to but never used

Remove assignment to unused variable `country`

(F841)